src/html.js

```diff
@@ -0,0 +1,142 @@
+import ToDoList from './toDo.js';
+import LocalStorage from './storage.js';
+
+export default class HtmlToDo {
+  // Edit mode
+  static toggleEditable(listElement) {
+    document.querySelectorAll('.toDoItem').forEach((element) => {
+      const edit = element.querySelector('.item-description');
+      const deleteBtn = element.querySelector('.delete-button');
+      const moveBtn = element.querySelector('.move-button');
+
+      if (element === listElement) {
+        deleteBtn.style.display = 'block';
+        moveBtn.style.display = 'none';
+        edit.readOnly = false;
+        element.classList.add('editing');
+      } else {
+        deleteBtn.style.display = 'none';
+        moveBtn.style.display = 'block';
+        edit.readOnly = true;
+        element.classList.remove('editing');
+      }
+    });
+  }
+
+  // Create list element
+  static createListItem(item) {
+    const listUl = document.querySelector('.toDoList');
+    const listElement = document.createElement('div');
+    listElement.classList.add('toDoItem');
+    listElement.classList.add('container-list');
+    listElement.style.order = item.index;
+
+    let crossed = '';
+    if (item.completed) {
+      crossed = 'crossed';
+    }
+
+    listElement.innerHTML = `
+     <input type="checkbox" class="checkbox">
+     <form class="edit-input"> <input type="text" name="title" class="item-description ${crossed}" value="${item.description}" readonly></form>
+     <button type="button" class="button move-button"><i class="fas fa-ellipsis-v"></i></button>
+     <button type="button" class="button delete-button"><i class="fas fa-trash-alt"></i></button>
+    `;
+
+    // Add  status changer
+    const completed = listElement.querySelector('.checkbox');
+    completed.checked = item.completed;
+    completed.addEventListener('change', (e) => {
+      ToDoList.changeStatus(item.index, e.target.checked);
+      listElement.querySelector('.item-description').classList.toggle('crossed');
+    });
+
+    // Add delete feature
+    const deleteBtn = listElement.querySelector('.delete-button');
+    deleteBtn.addEventListener('click', () => {
+      ToDoList.delete(item.index);
+      listElement.remove();
+    });
+
+    // Handle edit form
+    const editForm = listElement.querySelector('.edit-input');
+    editForm.addEventListener('submit', (e) => {
+      e.preventDefault();
+      const data = new FormData(editForm);
+      const newName = data.get('title');
+      ToDoList.update(newName, item.index);
+    });
```

Comment:
[OPTIONAL] I would use the event 'change' instead of 'submit' when the user edits the tasks. There is not indicative that to save a task you have to hit the `enter` key. Instead, with the 'change' event, whenever the user changes the state of the inputs the will be updated in localStorage. [You can read more about 'change' event here.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event)